### ISSUE TITLE
Unable to begin install on Raspberry Pi 4+ B

### ISSUE THREAD

**Issue by stratocaster21** [grokability/snipe-it#7649]
#### Expected Behavior (or desired behavior if a feature request)

I am attempting to install on a new Raspberry Pi 4+ B running Buster.

-----

#### Actual Behavior

When running install.sh I am greeted with the following info:

--2019-12-07 01:31:11-- https://raw.githubusercontent.com/snipe/snipe-it/master/snipeit.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.52.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.52.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 21737 (21K) [text/plain]
Saving to: ‘snipeit.sh’

snipeit.sh 0%[ ] 0 --.-KB/s
snipeit.sh 100%[===================>] 21.23K --.-KB/s in 0.01s

2019-12-07 01:31:11 (1.40 MB/s) - ‘snipeit.sh’ saved [21737/21737]

       _____       _                  __________
      / ___/____  (_)___  ___        /  _/_  __/
      \__ \/ __ \/ / __ \/ _ \______ / /  / /
     ___/ / / / / / /_/ /  __/_____// /  / /
    /____/_/ /_/_/ .___/\___/     /___/ /_/
                /_/

  Welcome to Snipe-IT Inventory Installer for CentOS, Fedora, Debian and Ubuntu!

  The installer was unable to determine your OS. Exiting for safety.

Nothing I am able to do will resolve the issue.

-----

#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode 
- [x] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade?
Fresh
- Version of Snipe-IT you're running
4.7.7
- Version of PHP you're running
7.3.11-1~deb10u1
- Version of MySQL/MariaDB you're running
mariadb-server-10.0.28.2
- What OS and web server you're running Snipe-IT on
Raspbian Buster
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
install.sh
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
n/a

If there's something I'm missing (high likelyhood) then please take pity on me,

**Comment by stratocaster21** [comment]
A log of the output right before it exits shows:

--2019-12-07 01:31:11--  https://raw.githubusercontent.com/snipe/snipe-it/master/snipeit.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.52.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.52.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 21737 (21K) [text/plain]
Saving to: ‘snipeit.sh’


snipeit.sh          0%[                    ]       0  --.-KB/s               
snipeit.sh        100%[===================>]  21.23K  --.-KB/s    in 0.01s   

2019-12-07 01:31:11 (1.40 MB/s) - ‘snipeit.sh’ saved [21737/21737]

**Comment by railworks2** [comment]
Hey there!

I found Snipe yesterday and tried the same but on my Raspberry Pi 2 (4 is currently in use) and I got to the same place with the same issue.

However, I've managed to fix this issue. Ubuntu currently offers Ubuntu Server for Raspberry Pi 2 to 4, I used Ubuntu 18.04.3 LTS as I found the latest version to cause issues. 

I hope that helps your issues and hopefully they can fix this issue for future users, but here is a workaround for now.
